docs_src/getting-started/Ch-GettingStartedDockerUsers.md

```diff
@@ -120,7 +120,8 @@ In a command terminal, change directories to the location of your docker-compose
 ``` bash
 docker-compose up -d
 ```
-
+!!! Warning
+    If you are using Docker Compose Version 2, please replace `docker-compose` with `docker compose` before proceeding. See:  <https://docs.docker.com/compose/install/compose-plugin/> for more information.
```

Comment:
Add a statement:  This should be applied to all the `docker-compose` commands in this website.

Use this link for more information: https://www.docker.com/blog/announcing-compose-v2-general-availability/ instead of the installation link

Reply:
udpated